packages/esm-patient-registration-app/src/patient-registration/field/address/address-field.component.tsx

```diff
@@ -74,7 +74,7 @@ export const AddressComponent: React.FC = () => {
   if (addressTemplate && !Object.keys(addressTemplate)?.length) {
     return (
       <AddressComponentContainer>
-        <SkeletonText role="progressbar" />
```

Comment:
This may be worth preserving as:

```tsx
<SkeletonText
  // @ts-expect-error
  role="progressbar"
/>
```

As with many components, the Carbon types here are somewhat incorrect as they actually [push all unknown properties to the underlying element](https://github.com/carbon-design-system/carbon/blob/6c5d4d0b146cf5c5221369296092532c451d7494/packages/react/src/components/SkeletonText/SkeletonText.tsx#L109), but [only declare the prop types they directly support in the type](https://github.com/carbon-design-system/carbon/blob/6c5d4d0b146cf5c5221369296092532c451d7494/packages/react/src/components/SkeletonText/SkeletonText.tsx#L20).